adc/consumer.py

```diff
@@ -121,11 +128,14 @@ def _stream_forever(self,
                         batch_size: int = 100,
                         batch_timeout: timedelta = timedelta(seconds=1.0),
                         ) -> Iterator[confluent_kafka.Message]:
-        while True:
+        self.running = True
+        while self.running is True:
```

Comment:
Replace all instances of `self.running is True` with `self.running` and all instances of `self.running is False` with `not self.running`.

Reply:
With use of an `Event` this would become `not self._stop. is_set()`, and similar for other uses.